common/buildings/wc_special_buildings.txt

```diff
@@ -42,5 +42,194 @@ dark_portal_02 = {
 		levy_size = 0.15
 	}
 
+	type = special
+}
+
+########
+# Gurubashi Arena
+
+gurubashi_arena_01 = {
+	type_icon = "icon_structure_colosseum.dds"
+	
+	character_modifier = {
+		martial = 5
+		levy_reinforcement_rate = 1
+		knight_limit = 2
+		knight_effectiveness_mult = 0.3
+	}
+
+	type = special
+}
+
+########
+# Karazhan
+
+karazhan_01 = {
+	type_icon = "icon_structure_brihadeeswarar_temple.dds"
+	
+	province_modifier = {
+		hostile_raid_time = 4
+		fort_level = 6
+	}
+
+	character_modifier = {
+		learning = 3
+		monthly_lifestyle_xp_gain_mult = 0.4
+		monthly_piety_gain_mult = 0.2
+		caster_toughness_mult = 0.7
```

Comment:
I would nerf it... looks too strong. Holy places give much less.

Reply:
I would say other buildings need nerf too. If you use the balance spreedsheet, I should aim at lesser cost number.

Reply:
40-30 prob.